src/schema/v2/artwork/index.ts

```diff
@@ -704,6 +704,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ arta_enabled }) => arta_enabled,
       },
+      artsyShippingInternational: {
```

Comment:
I'm wondering if we also need to change the [shipping info string ](https://github.com/artsy/metaphysics/pull/3855/files#diff-54b84e0c164089fc202e3439d8ded89ab4925442fb613f4e36a9c22acdb66cb8L760)
We could go with "Calculated in checkout" when it's either `process_with_arta_shipping` or this, what do you think?

Reply:
Ooo good catch, didnt realize that was happening in MP. Will adjust